src/java.base/share/classes/java/lang/invoke/VarHandles.java

```diff
@@ -657,6 +657,14 @@ private static boolean isCheckedException(Class<?> clazz) {
                 !Error.class.isAssignableFrom(clazz);
     }
 
+    private static boolean byteToBoolean(byte b) {
+        return b != 0;
+    }
+
+    private static byte booleanToByte(boolean b) {
+        return b ? (byte)1 : (byte)0;
+    }
+
```

Comment:
I think these can be removed as well now?

Reply:
right, removed